docs/manual/install/index.rst

```diff
@@ -27,8 +27,18 @@ Installing From Source
 First, you need to install all of Qtile's dependencies (although some are
 optional/not needed depending on your Python version, as noted below).
 
-Note that Python 3 versions 3.5 and newer are currently supported and tested,
-including corresponding PyPy3 versions.
+Note that Python 3 versions 3.7, 3.8, and 3.9 are currently supported and
+tested, including corresponding PyPy3 versions(currently only PyPy3.7). There
```

Comment:
I wonder if we shouldn't just put a link to the right place in `.github/workflows/ci.yaml` vs. explicitly listing versions here since things will inevitably get out of date again.

Reply:
That's fair, I'll do it in a bit.

Reply:
done!

Reply:
I linked to the folder, since the multiple ci workflows PR changes this.